Gemfile

```diff
@@ -28,6 +28,7 @@ group :test do
   gem 'voxpupuli-release',                                          :require => false, :git => 'https://github.com/voxpupuli/voxpupuli-release-gem.git'
   gem 'puppet-strings',                                             :require => false, :git => 'https://github.com/puppetlabs/puppetlabs-strings.git'
   gem 'rubocop', '~> 0.39',                                         :require => false
+  gem 'listen', '3.0.5'
```

Comment:
why do you add the listen gem? it is already present in the development group. (at least sort of, we've guard-rake there which requires listen)